src/plugins/discover/public/application/main/services/discover_data_state_container.ts

```diff
@@ -149,20 +146,20 @@ export function getDataStateContainer({
    * to be processed correctly
    */
   const refetch$ = new Subject<DataRefetchMsg>();
-  const shouldSearchOnPageLoad =
-    uiSettings.get<boolean>(SEARCH_ON_PAGE_LOAD_SETTING) ||
-    getSavedSearch().id !== undefined ||
-    !timefilter.getRefreshInterval().pause ||
-    searchSessionManager.hasSearchSessionIdInURL();
-  const initialFetchStatus = shouldSearchOnPageLoad
-    ? FetchStatus.LOADING
-    : FetchStatus.UNINITIALIZED;
+  const getInitialFetchStatus = () => {
```

Comment:
When SEARCH_ON_PAGE_LOAD_SETTING is off, it still loads data on page refresh and after pressing New

![Mar-27-2023 14-41-09](https://user-images.githubusercontent.com/1415710/227950678-28e26639-23d9-4548-94d1-6cfdfc510c04.gif)

Reply:
Working on a fix, moving the listener for the `/` path, that creates a new saved search or resets an existing one to `discover_main_route.tsx`

https://github.com/elastic/kibana/pull/153528/commits/5ba1e2b1ed786d5beb538ed87fcad92d7a4c21b5

The way it was implemented before caused a duplicate fetch command before ... will follow up, first CI needs to reply

Reply:
The issue is still present

Reply:
@davismcphee I can't reproduce it, but this needs definitely another pair of 👀

Reply:
Well, well, found and fixed it (needed a query to reproduce) https://github.com/elastic/kibana/pull/153528/commits/98efa69830cb63ad307bee3943b2eba9f14949b2

Reply:
thx!

Reply:
Looks like this issue is happening again

Reply:
We chatted "offline" and it was because the refresh interval was "On" ... So it works like before. However me might consider, switching the refresh interval to "Off" when triggering `New` ... When I start a new search I think I first want to apply data, query, filter, before I want to have the data refreshed?